aspnetcore/test/troubleshoot-azure-iis/includes/troubleshoot-azure-iis2.md

```diff
@@ -401,7 +401,7 @@ Setting the environment variable for `ASPNETCORE_ENVIRONMENT` is only recommende
 
 If an app is capable of responding to requests, obtain request, connection, and additional data from the app using terminal inline middleware. For more information and sample code, see <xref:test/troubleshoot#obtain-data-from-an-app>.
 
-### Slow or hanging app (IIS)
+### Slow or nonresponding app (IIS)
```

Comment:
```suggestion
### Slow or non-responding app (IIS)
```

Reply:
Not per Chicago.

Reply:
Very minor, take or leave: I think you can go either way technically, but I think non-responding might be a bit more common.

Reply:
"non-" is listed ...

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/non

... and we defer to Chicago per MS guidelines ...

https://learn.microsoft.com/en-us/style-guide/grammar/grammar-and-parts-of-speech